docs/errors/usage_errors.md

```diff
@@ -1188,3 +1186,27 @@ try:
 except PydanticUserError as exc_info:
     assert exc_info.code == 'dataclass-on-model'
 ```
+
+## [`Unpack`][typing.Unpack] used without a [`TypedDict`][typing.TypedDict] {#unpack-typed-dict}
+
+This error is raised when [`Unpack`][typing.Unpack] is used with something else than
+a [`TypedDict`][typing.TypedDict] class object to type hint variadic keyword arguments.
+
+For reference, see the [related specification section] and [PEP 692].
+
+```py
+from typing_extensions import Unpack
+
+from pydantic import validate_call
+
+try:
+
```

Comment:
Do we need this whitespace here for linting?

Reply:
Yes because of the function definition